app/views/DelayTypeView.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton,
+        h1: components.h1,
+        p: components.p
+)
+
+@(form: Form[_], mode: Mode)(implicit request: DataRequest[_], messages: Messages)
+
+    @layout(pageTitle = title(form, messages("delayType.title"))) {
+
+        @formHelper(action = routes.DelayTypeController.onSubmit(request.ern, request.arc, mode), 'autoComplete -> "off") {
+
+            @if(form.errors.nonEmpty) {
+                @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
+            }
+
+            @h1("delayType.heading", Some(messages("arc.subHeading", request.arc)), "govuk-heading-l")
+
+            @p(classes = "govuk-body") {
+                @messages("delayType.p1")
+            }
+
+            @p(classes = "govuk-body") {
+                @messages("delayType.p2")
+            }
+
+            @govukRadios(
+                RadiosViewModel(
+                    field = form("value"),
+                    legend = LegendViewModel(HtmlContent(messages("delayType.title"))).withCssClass("govuk-fieldset__legend--m"),
+                    items = DelayType.options
+                )
+            )
+
+            @govukButton(
+                ButtonViewModel(messages("site.continue"))
+            )
+        }
+    }
+
+    @{
+        //$COVERAGE-OFF$
+    }
```

Comment:
```suggestion
@layout(pageTitle = title(form, messages("delayType.title"))) {

    @formHelper(action = routes.DelayTypeController.onSubmit(request.ern, request.arc, mode), 'autoComplete -> "off") {

        @if(form.errors.nonEmpty) {
            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
        }

        @h1("delayType.heading", Some(messages("arc.subHeading", request.arc)), "govuk-heading-l")

        @p(classes = "govuk-body") {
            @messages("delayType.p1")
        }

        @p(classes = "govuk-body") {
            @messages("delayType.p2")
        }

        @govukRadios(
            RadiosViewModel(
                field = form("value"),
                legend = LegendViewModel(HtmlContent(messages("delayType.title"))).withCssClass("govuk-fieldset__legend--m"),
                items = DelayType.options
            )
        )

        @govukButton(
            ButtonViewModel(messages("site.continue"))
        )
    }
}

@{
    //$COVERAGE-OFF$
}
```

Indentation is a bit weird here - just needs moving back 1

Reply:
Updated